docs/en/stack/ml/anomaly-detection/anomaly-examples.asciidoc

```diff
@@ -18,6 +18,7 @@ The scenarios in this section describe some best practices for generating useful
 * <<ml-configuring-populations>>
 * <<ml-configuring-transform>>
 * <<ml-delayed-data-detection>>
+* <<ml-configuring-alerts>>
```

Comment:
> * <<ml-configuring-alerts>>

I don't think there's an order to this list other than alphabetical. In which case, we need to sort the last two items in this list.